doc/install.rst

```diff
@@ -1,30 +1,31 @@
 Installation
-------------
-.. warning::
-   Ensure that the :ref:`pre-requisites` from the previous section
-   have been performed, checked and tested before proceeding.
+============
 
-.. note:: The library has been tested against Python 2.7, 3.4, 3.5 and 3.6.
+The successful installation of a display module to your SBC requires a combination
+of tasks to be completed before the display will operate correctly.
 
-   For **Python3** installation, substitute the following in the
-   instructions below.
+First, the device needs to be wired up correctly to your single-board computer
+(SBC) and the interface that will be used needs to be enabled in the kernel
+of the operating system of the SBC.  Instructions to for this are provided in
+:doc:`hardware`.
 
-   * ``pip`` ⇒ ``pip3``,
-   * ``python`` ⇒ ``python3``,
-   * ``python-dev`` ⇒ ``python3-dev``,
-   * ``python-pip`` ⇒ ``python3-pip``.
+Equally important, the ``luma.oled`` software needs to be installed including
+the build dependencies that for the python modules it uses.  Instructions
+to complete that task are provided in :doc:`software`.
 
-   It was *originally* tested with Raspbian on a rev.2 model B, with a vanilla
-   kernel version 4.1.16+, and has subsequently been tested on Raspberry Pi
-   (both Raspbian Jessie and Stretch) models A, B2, 3B, Zero, Zero W and
-   OrangePi Zero (Armbian Jessie).
+Finally, you need to leverage the appropriate interface class and display
+class for your device to implement your application.  Instructions for that
+are included in :doc:`python-usage`.
 
-Install the latest version of the library directly from PyPI_::
+.. note:: This library has been tested against Python 3.5, 3.6, 3.7 and 3.8.
 
-  $ sudo apt install python-dev python-pip libfreetype6-dev libjpeg-dev build-essential libopenjp2-7 libtiff5
-  $ sudo -H pip install --upgrade luma.oled
+   It was *originally* tested with Raspbian on a rev.2 model B, with a vanilla
+   kernel version 4.1.16+, and has subsequently been tested on Raspberry Pi
+   (Raspbian Jessie, Stretch and Buster) models A, B2, 3B, Zero, Zero W,
```

Comment:
also add pi 4

Reply:
Reworded paragraph to make it clearer which hardware and OS versions had been tested.